index.html

```diff
@@ -26,11 +26,29 @@ <h1 class="newsletter-title">Vue.js Newsletter</h1>
           </div>
         </form>
       </div>
-      <div class="newsletter__previous"><a href="/issues.html">Browse list of previous issues</a></div>
+
+      <div class="content">
+        <div class="content-read">
+            <p class="content-title">Read the News</p>
+            <div class="newsletter__previous"><a href="/issues.html">Browse previous issues</a></div>
```

Comment:
I'm not sure why there was an old and outdated link for previous issues subpage (at this moment we redirect to `https://www.getrevue.co/profile/vuenewsletter` for up-to-date list of issues and I think we should stick to it).